src/test/java/io/github/nscuro/versatile/VersTest.java

```diff
@@ -86,6 +92,34 @@ void testSimplify(final String before, final String after) {
         assertThatNoException().isThrownBy(vers::validate);
     }
 
+    private static Stream<Arguments> testSplitVersArguments() {
+        return Stream.of(
+                arguments(
+                        "vers:generic/1.2.3|>=2.0.0|<5.0.0",
+                        List.of("vers:generic/1.2.3", "vers:generic/>=2.0.0|<5.0.0")
+                ),
+                arguments(
+                        "vers:pypi/>0.0.0|>=0.0.1|0.0.2|<0.0.3|0.0.4|<0.0.5|>=0.0.6",
+                        List.of("vers:pypi/>0.0.0|<0.0.5", "vers:pypi/>=0.0.6")
+                ),
+                arguments(
+                        "vers:pypi/>0.0.0|>=0.0.1|0.0.2|0.0.3|0.0.4|<0.0.5|>=0.0.6|!=0.8",
+                        List.of("vers:pypi/>0.0.0|<0.0.5", "vers:pypi/!=0.8", "vers:pypi/>=0.0.6")
+                ),
+                arguments(
+                        "vers:npm/>=1.0.0-beta1|<=1.7.5|>=7.0.0-M1|<=7.0.7|>=7.1.0|<=7.1.2|>=8.0.0-M1|<=8.0.1",
+                        List.of("vers:npm/>=1.0.0-beta1|<=1.7.5", "vers:npm/>=7.0.0-M1|<=7.0.7", "vers:npm/>=7.1.0|<=7.1.2", "vers:npm/>=8.0.0-M1|<=8.0.1")
+                )
```

Comment:
A few more tests would be good here.

e.g. what happens if there is only a single constraint? What happens if the only constraint is a wildcard (`*`)?